starter-code/apple-pie-starter-code/index.html

```diff
@@ -2,34 +2,47 @@
 <html lang="en">
 <head>
   <title>Apple Pie Recipe</title>
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,600" rel="stylesheet">
+  <link href="css/main.css" rel="stylesheet">
 </head>
 <body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
-quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
+  <div class="header">
+      <h1>Apple Pie</h1>
+  </div>
 
-Ingredients
-1 recipe pastry for a 9 inch double crust pie
-1/2 cup unsalted butter
-3 tablespoons all-purpose flour
-1/4 cup water
-3 tablespoons all-purpose flour
-1/4 cup water
-1/2 cup white sugar
-1/2 cup packed brown sugar
-8 Granny Smith apples - peeled, cored and sliced
+  <div class="content">
+      <p>
+          This was my grandmother's apple pie recipe. I have never seen another one
+          quite like it. It will always be my favorite and has won me several first place
+          prizes in local competitions. I hope it becomes one of your favorites as well!
+      </p>
 
-Directions
-Preheat oven to 425 degrees F (220 degrees C). Melt the butter in a saucepan. Stir in
-flour to form a paste. Add water, white sugar and brown sugar, and bring to a boil. Reduce
-temperature and let simmer.
+      <div class="ingredients">
+          <h3>Ingredients</h3>
+          <ul>recipe pastry for a 9 inch double crust pie</ul>
```

Comment:
Be careful this is not how you use the list tag, the ul is here to say to the browser that this is an unordered list (the opposite of an ol, ordered list), you should use the list tag like this :  

```html 

<ul>
   <li>element 1</li>
   <li>element 2</li>
</ul>

<ol>
   <li>element 1</li>
   <li>element 2</li>
</ol>

```